src/org/opendatakit/aggregate/parser/BaseFormParserForJavaRosa.java

```diff
@@ -111,12 +116,29 @@ private static final void initializeJavaRosa() {
              // new CoreModelModule().registerModule();
              // replace with direct call to PrototypeManager
              PrototypeManager.registerPrototypes(SERIALIABLE_CLASSES);
+             redirectOutput();
              new XFormsModule().registerModule();
              isJavaRosaInitialized = true;
        }
     }
   }
 
+  private static void redirectOutput() {
+    File jrLogFile = new File(FileSystemUtils.getBriefcaseFolder(), ".briefcase-javarosa.log");
+    try {
+      final PrintStream jrOut = new PrintStream(jrLogFile);
+      Std.setOut(jrOut);
+      Std.setErr(jrOut);
+      Runtime.getRuntime().addShutdownHook(new Thread() {
+        public void run() {
+          jrOut.close();
+        }
+      });
+    } catch (FileNotFoundException e) {
+      log.warn("failed to redirect javarosa output to " + jrLogFile);
```

Comment:
> reflect the fact that this impacts more than JavaRosa output.
...but it doesn't. This class is scoped to JavaRosa as are `setOut` and `setErr`. If we're using `Std` elsewhere, we shouldn't be. This is literally just to redirect the output that JavaRosa generates using the Std class, nothing more.